### PR TITLE
Add reference to using apache/daffodil-schema.g8

### PR DESCRIPTION
**Pull request by stevedlawrence**

### CODE REVIEW

File: site/dfdl-layout.md [hunk 1]
```diff
@@ -49,8 +65,6 @@ These conventions are actually usable for regular XML-schema projects, that is,
 they're not really DFDL-specific conventions. They're general conventions for
 organizing projects so as to achieve the above benefits.
 
-### Conventions
-
```
Comment (tuxji) [inline site/dfdl-layout.md:53]: Although you've moved the Conventions header up a few paragraphs, I think the following paragraphs belong more to a new section than the "Conventions" section.  Should we insert a new header like "Standard File Tree" or "Standard Project Layout" after line 67?

**Review by tuxji — APPROVED** [review]
+1

File: site/dfdl-layout.md (hunk 1 above)
Comment (stevedlawrence) [reply site/dfdl-layout.md:53]: Agreed, new fixupc commit added.

**Comment by tuxji** [comment]
@stevedlawrence I think you forgot this pull request is ready to be squashed and merged :).

**Pull request merged**